libs/parallelism/algorithms/tests/regressions/reduce_3641.cpp

```diff
@@ -8,24 +8,21 @@
 // #3641: Trouble with using ranges-v3 and hpx::parallel::reduce
 // #3646: Parallel algorithms should accept iterator/sentinel pairs
 
-#include <hpx/hpx_main.hpp>
```

Comment:
Please don't remove this header. Including it ensures that `main` is started on the HPX runtime.